spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Helper.scala

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
I think other similar class are named `Util` ?

Reply:
Updated. Renamed to `Utils` now.